index.js

```diff
@@ -310,6 +310,20 @@ module.exports = {
       return pull(ssb.box2.listGroupIds({ live: !!opts.live }), paraMap(get, 4))
     }
 
+    function getRootIdOfMsgAuthor(groupRootMsgId, cb) {
```

Comment:
move into metafeed helper file

Reply:
ah i guess we don't have that file yet https://github.com/ssbc/ssb-tribes2/pull/59